link/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignup.kt

```diff
@@ -196,70 +202,86 @@ internal fun LinkInlineSignup(
                     }
 
                     AnimatedVisibility(
-                        visible = expanded,
-                        modifier = Modifier.padding(horizontal = 16.dp)
+                        visible = expanded
                     ) {
-                        Column(
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .padding(bottom = 16.dp)
-                        ) {
-                            EmailCollectionSection(
-                                enabled = enabled,
-                                emailController = emailController,
-                                signUpState = signUpState,
-                                focusRequester = focusRequester
-                            )
-
-                            AnimatedVisibility(
-                                visible = signUpState != SignUpState.InputtingPhoneOrName &&
-                                    errorMessage != null
+                        Column {
+                            Divider()
```

Comment:
These dividers look lighter in the mocks. Since we need to match the appearance API, can you make their color
`MaterialTheme.paymentsColors.componentBorder.copy(alpha = 0.1f)`